packages/calculator/src/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Main from './main';
+
+export default class Calculator extends HTMLElement {
+
+	constructor() {
```

Comment:
can you use 2 spaces (not tabs) indentation? I've added this to an `.editorconfig` file